### ISSUE TITLE
Creation of README.md (Japanese version), etc.

### ISSUE THREAD

**Issue by hope366** [yzyray/lizzieyzy#251]
I noticed that the Japanese explanation was missing in README.md, so I created it today.
I wanted to design it to open in a new tab when opening a link, but it didn't work.

  `<a href="http://example.com/" target="_blank">Hello, world!</a>`
 `[link](url){:target="_blank"}`

I did a little research and tried the two mentioned above, but neither worked.
I looked at some other repositories, but none of the links were designed to open in a new tab. Is that what GitHub is?

**Comment by LatinScribe** [comment]
You're right - this is a Github thing - since README.md is a markdown file, and the GitHub markdown parser does not support the target attribute (even in plain HTML). See this stack overflow post: https://stackoverflow.com/questions/41915571/open-link-in-new-tab-with-github-markdown-using-target-blank.

**Comment by hope366** [comment]
Thank you for providing us with useful information.
It's a bit of a shame that this feature isn't available, but many browsers have an "open in new tab" feature in the right-click menu, so it doesn't really seem to be a big deal.
